packages/api/src/services/microsoft.calendar.ts

```diff
@@ -160,32 +171,35 @@ export class MicrosoftCalendarService {
         emailAddress: { address: email, name: email },
         type: "required",
       })),
-      singleValueExtendedProperties: [
-        {
-          id: EXTENDED_PROP_SEARCH,
-          value: "true",
-        },
-        {
-          id: EXTENDED_PROP_ID_SEARCH,
-          value: id,
-        },
-      ],
+      // singleValueExtendedProperties: [
+      //   {
+      //     id: EXTENDED_PROP_SEARCH,
+      //     value: "true",
+      //   },
+      //   {
+      //     id: EXTENDED_PROP_ID_SEARCH,
+      //     value: id,
+      //   },
+      // ],
```

Comment:
needs testing with business account, my understanding is we let microsoft create the id for us, so why do we need to send ours?

Reply:
@bgavas do you know why we're sending our id along?